### PR TITLE
Exercise

### PR DESCRIPTION
**Pull request by pob31**
listed all primitives of C++
added value declarations according to exercise
created 10 functions with their calls

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -65,10 +63,18 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
+    float floatNumber = 3.4f;
+    char charVariable{'c'};
+    bool boolValue = true;
+    unsigned int absoluteValue = 3;
+    double bigFloat = -123456789.23456;
```
Comment (matkatmusic) [inline main.cpp:70]: instructions ask for 3 instances of each type.
- [ ] revise according to spec

File: main.cpp [hunk 2]
```diff
@@ -65,10 +63,18 @@ void variableDeclarations()
...
     
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(floatNumber);
+    ignoreUnused(charVariable);
+    ignoreUnused(boolValue);
+    ignoreUnused(absoluteValue);
+    ignoreUnused(bigFloat);
```
Comment (matkatmusic) [inline main.cpp:77]: - [ ] pass them all to a single call of `ignoreUnused`

File: main.cpp [hunk 3]
```diff
@@ -81,72 +87,130 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+  return(x>y);
+}
 /*
- 10)
+ 10) the last one
  */
+int theLastOne()
+{
+  return 10;
+}
```
Comment (matkatmusic) [inline main.cpp:174]: You need to set your repl.it indent according to the instructions for the coding standard here: 
https://github.com/pob31/PFMCPP_Project2/blob/master/README.md#5a-make-a-branch-in-replit
- [ ] fix indentation
- [ ] fix spacing around operators.
Make your code *look* good.

File: main.cpp [hunk 4]
```diff
@@ -81,72 +87,130 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    std::cout << std::endl << "5: I say true but you never agree: "<< neverAgree(true)<< std::endl;
     //6)
-    
+    playDrone(50.0f,4000.0f);
     //7)
-    
+    std::cout << std::endl << "7 : Please type a letter and press return afterwards: ";
+    char pressKey;
+    std::cin>>pressKey;
+    typeLetter(pressKey);
```
Comment (matkatmusic) [inline main.cpp:205]: - [ ] don't use std::cin in any course projects, please

File: main.cpp (hunk 1 above)
Comment (pob31) [reply main.cpp]: 3 declaration for each type.

File: main.cpp (hunk 2 above)
Comment (pob31) [reply main.cpp]: Single call of ignoreUnused for several variables at once.

File: main.cpp (hunk 3 above)
Comment (pob31) [reply main.cpp]: Cleaned up the layout.

File: main.cpp (hunk 4 above)
Comment (pob31) [reply main.cpp]: removed std::cin

**Pull request merged**